### PR TITLE
Notifications - Adapting to new design

### PR DESCRIPTION
**Pull request by tonysaad**
Removing font from here and just use it from `scss` according to this PR https://github.com/wholesale-design-system/scss/pull/26

screen shot 2017-03-29 at 16 08 34

### CODE REVIEW

File: src/ws-notification/ws-notification.js [hunk 1]
```diff
@@ -123,16 +116,13 @@ export class WSNotification extends Component {
   close(index) {
     const notification = this.refs[`notification-${index}`];
     if (notification) {
-      const notifications = this.state.notifications.splice(index, 1);
-      notification.style.transition = 'opacity .2s ease-out, max-height .8s ease, margin-bottom .8s ease';
-      notification.style.willChange = 'opacity, max-height, margin-bottom';
-      notification.style.opacity = 0;
-      notification.style.maxHeight = 0;
-      notification.style.marginBottom = '-4.5rem';
```
Comment (fragsalat) [inline src/ws-notification/ws-notification.js]: Why did you remove the animations?

File: src/ws-notification/ws-notification.scss [hunk 2]
```diff
@@ -1,13 +1,121 @@
...
-  bottom: 0;
-}
-.ws-notification-list {
-  margin: $space-default;
-}
-.ws-notification-list .notification {
-  display: block;
-  margin: $space-stack-m;
-  max-height: 300px;
+$export-notification-css: true !default;
```
Comment (fragsalat) [inline src/ws-notification/ws-notification.scss]: You don't need this anymore^^

Comment (tonysaad) [inline src/ws-notification/ws-notification.scss]: True :)
